### ISSUE TITLE
Add [Sweat Grapes]

### ISSUE THREAD

**Issue by eifetx** [google/fonts#5842]

**Comment by applecuckoo** [comment]
Hi there @eifetx! Do you have the permission of the author to put this on Google Fonts? Putting the files through [Wakamai Fondue](https://wakamaifondue.com/beta/) shows that this seems to be a commercial font.
![Screenshot 2023-01-31 at 10-16-21 Wakamai Fondue](https://user-images.githubusercontent.com/113647417/215597668-76c207ac-b371-4b1c-82fa-ba3528396096.png)

**Comment by eifetx** [comment]
> Hi there @eifetx! Do you have the permission of the author to put this on Google Fonts? Putting the files through [Wakamai Fondue](https://wakamaifondue.com/beta/) shows that this seems to be a commercial font. ![Screenshot 2023-01-31 at 10-16-21 Wakamai Fondue](https://user-images.githubusercontent.com/113647417/215597668-76c207ac-b371-4b1c-82fa-ba3528396096.png)

hello. first, im new here. i dont know how it works. sorry i dont edited a font. im just want to try ( my font Sweaty Grapes ) in my google fonts, but i dont know. how put in there.

**Comment by applecuckoo** [comment]
Oh, I was just asking if you made this font. Since you made it, that's fine, though I don't work for Google Fonts, I'll just let one of the staff take over and start the process of adding Sweaty Grapes. 👌🏻

**Comment by eifetx** [comment]
@davelab6 hello dave, i just got email from you. you want to discuss with this font ?

**Comment by davelab6** [comment]
Yes, as part of your overall collection of submissions. Overall I think your designs are very promising, so I'm eager to chat with you and learn more about your journey into type design :)

**Comment by eifetx** [comment]
@davelab6 hello dave, i just join your call in google meet. but didn't answer

**Comment by davelab6** [comment]
@eifetx I am happy to say that @emmamarichal will work with you again this year to onboard this one :) Great work!

And thanks for emailing your story, its a great one!

**Comment by emmamarichal** [comment]
I don't see any repo anymore, @eifetx let us know if you still plan to release it :) thanks!
